File: Bank/EntitiesVb/Account.vb
Public Class Account
    Public Sub New()

    End Sub
    Public Sub New(number As Integer, balance As Decimal)
        Me.Number = number
        _balance = balance
    End Sub
    Public Property Number As Integer
    Public Property Enabled As Boolean
    Private _balance As Decimal
    Public ReadOnly Property Balance As Decimal
        Get
            Return _balance
        End Get
    End Property
    Public Property Customer As Customer

    Public Sub Deposit(value As Decimal)
        plusBalance(value)
    End Sub
    Public Sub Withdraw(value As Decimal)
        plusBalance(-value)
    End Sub
    Private Sub plusBalance(value As Decimal)
        _balance = _balance + value
    End Sub
End Class

File: Bank/EntitiesCs/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EntitiesCs
{
    public class Account
    {
        private Decimal _balance = 0;
        public Account()
        {
        }
        public Account(int number, decimal balance)
        {
            Number = number;
            _balance = balance;
        }
        public int Number { get; set; }
        public bool Enabled { get; set; }
        public Decimal Balance { get => _balance; }
        public void Deposit(Decimal value)
        {
            plusBalance(value);
        }
        public void WithDraw(Decimal value)
        {
            plusBalance(-value);
        }
        private void plusBalance(decimal value)
        {
            _balance += value;
        }
    }
}
